mujtaba-breakout-room/jacki-chan/style.css

```diff
@@ -0,0 +1,199 @@
+    height: 150px;
+    border: 2px solid white;
+    display: inline-block;
+}
+
+.logo {
+    width: 50px;
+    height: 50px;
+    margin-left: 8px;
+}
+
+.header-right {
+    margin-top: 16px;
+    margin-right: 8px;
+    float: right;
+}
+
+.btn-white {
+    color: #0277bd;
+    border: 1px solid #0277bd;
+    background-color: white;
+    border-radius: 5px;
+    padding: 8px 16px;
+    text-transform: uppercase;
+}
+
+.btn-blue {
+    color: white;
+    border: 1px solid #0277bd; 
+    background-color: #039be5;
+    border-radius: 5px;
+    padding: 8px 16px;
+    text-transform: uppercase;
+    margin-left: 16px;
+}
+
+.btn-white:hover {
+    background-color: #eeeeee;
+}
+
+.btn-blue:hover {
+    background-color: #0277bd;
+}
+.btn-white2{
+    color: #0277bd;
+    border: 1px solid #0277bd;
+    background-color: white;
+    border-radius: 5px;
+    padding: 8px 16px;
+    text-transform: uppercase;
+    margin-left: 10px;
+    margin-right: 10px;
+    position: relative;
+    top: 30px;
+
+}
+.btn-blue2 {
+    color: white;
+    border: 1px solid #0277bd; 
+    background-color: #039be5;
+    border-radius: 5px;
+    padding: 8px 16px;
+    text-transform: uppercase;
+    margin-left: 16px;
+    position: relative;
+    top: 30px;
+}
+#horLine{
+    position: relative;
+    top: 50px;
+    left: 80px;
+    width: 80%;
+}
+.personalInfo{
+    margin-left: 100px;
+    margin-top: 90px;
+}
+h4 ~ h4{
+    color: gray;
+    position: relative;
+    left: 20px;
+    bottom: 18px;
+}
+.followersImg{
+    width: 30px;
+   height: 30px;
+   border-radius:50%;
+}
+.followers{
+    position: relative;
+    left: -20px;
+}
+.followersImg1{
+    width: 30px;
+   height: 30px;
+   border-radius:50%;
+   border: 3px solid white ;
+}
+.followersImg2{
+    width: 30px;
+   height: 30px;
+   border-radius:50%;
+   border: 3px solid white ;
+   position: relative;
+    left: -18px;
+}
+.followersImg3{
+    width: 30px;
+   height: 30px;
+   border-radius:50%;
+   border: 3px solid white ;
+   position: relative;
+    left: -36px;
+}
+.followersImg4{
+    width: 30px;
+   height: 30px;
+   border-radius:50%;
+   border: 3px solid white ;
+   position: relative;
+    left: -54px;
+}
+.followersImg5{
+    width: 30px;
+   height: 30px;
+   border-radius:50%;
+   border: 3px solid white ;
+   position: relative;
+    left: -72px;
+}
+.followersImg6{
+    width: 30px;
+    height: 30px;
+    border-radius:50%;
+    border: 3px solid white ;
+    position: relative;
+     left: -90px;
+}
+.followersImg7{
+    width: 30px;
+   height: 30px;
+   border-radius:50%;
+   border: 3px solid white ;
+   position: relative;
+    left: -108px;
+}
+.followersImg8{
+    width: 30px;
+    height: 30px;
+    border-radius:50%;
+    border: 3px solid white ;
+    position: relative;
+     left: -126px;
+}
+.followersImg9{
+    width: 30px;
+    height: 30px;
+    border-radius:50%;
+    border: 3px solid white ;
+    position: relative;
+     left: -144px;
+}
```

Comment:
Instead of adding the images using the relative position and decreasing the left value with each new image and having a class for each image, you can do this by simply giving a fixed left margin to each image.

Like this:

.followers img{
    width: 30px;
    height: 30px;
    border-radius: 50%;
    border: 3px solid white;
    margin-left:-15px;
}

-15px so that the images can overlap each other.